crates/ruff_python_trivia/src/tokenizer.rs

```diff
@@ -71,23 +71,19 @@ pub fn lines_after(offset: TextSize, code: &str) -> u32 {
 }
 
 /// Returns the position after skipping any trailing trivia up to, but not including the newline character.
-pub fn skip_trailing_trivia(offset: TextSize, code: &str) -> TextSize {
-    let tokenizer = SimpleTokenizer::starts_at(offset, code);
-
-    for token in tokenizer {
-        match token.kind() {
-            SimpleTokenKind::Whitespace
-            | SimpleTokenKind::Comment
-            | SimpleTokenKind::Continuation => {
-                // No op
-            }
-            _ => {
-                return token.start();
-            }
-        }
-    }
-
-    offset
+pub fn after_trailing_trivia(offset: TextSize, code: &str) -> TextSize {
```

Comment:
I'm unsure, and maybe the name is still unclear. I thought about that too when commenting. 

This returns the position of the line end, or the start position of the next non-trivia token. The second case doesn't seem to matter for all usages. 

Since we don't know the answer, should we change the method to `lines_after_ignoring_trivia` or similar, because we have no need for the more generic case? This also allows counting the newline tokens (skip while not a new line and trivia, then take while newline, count)

Reply:
Ah I see, we always call `lines_after`. Ok, got it, will change.